server/src/controllers/deleteBooking.js

```diff
@@ -0,0 +1,8 @@
+const { deleteBookingById } = require('../database/queries');
+
+module.exports = (req, res, next) => {
+  const { id } = req.params;
+  deleteBookingById(id)
```

Comment:
this means that any user can delete any booking from the system, which is wrong, figure out a way to make it either the admin or the user who created this booking